lib/vintage_net_wizard/application.ex

```diff
@@ -5,11 +5,10 @@ defmodule VintageNetWizard.Application do
 
   def start(_type, _args) do
     backend = Application.get_env(:vintage_net_wizard, :backend, VintageNetWizard.Backend.Default)
-    port = Application.get_env(:vintage_net_wizard, :port, 80)
 
     children = [
-      {VintageNetWizard.Backend, [backend]},
-      {VintageNetWizard.Web.Endpoint, [port]}
+      {VintageNetWizard.Web.Endpoint, []},
+      {VintageNetWizard.Backend, [backend]}
```

Comment:
Doesn't the `Web.Endpoint` make calls into the `Backend`, so shouldn't it be started second?

Reply:
No, the web endpoint is now a dynamic supervisor, so it won't make calls until a child is started, which is started by the backends.

Reply:
Ok, I see. Then why did you reorder them?

Reply:
Because the backends start the server now, so the supervisor that runs the server needs to be started first. The naming might be odd so it's completely clear. I figured since `Endpoint` was already a supervisor, I would just it to be dynamic, and then provide an API for stopping and starting the server part of it.